abr_control/interfaces/vrep.py

```diff
@@ -327,3 +327,21 @@ def set_xyz(self, name, xyz):
             -1,  # set absolute, not relative position
             xyz,
             vrep.simx_opmode_blocking)
+
+    def get_sim_time(self):
```

Comment:
Do you need this function?

Reply:
don't think so. i've been on branches without and not run into it